main.go

```diff
@@ -25,34 +38,36 @@ func main() {
 		panic(err.Error())
 	}
 
+	fmt.Println("Beginning the crawl.")
+
 	for {
 		pods, err := clientset.CoreV1().Pods("").List(metav1.ListOptions{})
 		if err != nil {
 			panic(err.Error())
 		}
 
-		restartThreshold := int32(100)
+		restartThreshold := int32(NUMBER_RESTARTS)
 
 		for _, item := range pods.Items {
 			for _, status := range item.Status.ContainerStatuses {
 				waiting := status.State.Waiting
 
 				if waiting != nil {
 					reason := waiting.Reason
-					if reason == "CrashLoopBackOff" && status.RestartCount > restartThreshold {
+					if stringInSlice(reason, waitingReasons) && status.RestartCount > restartThreshold {
 						fmt.Println(item.Namespace + "/" + item.Name + " has " +
```

Comment:
I would convert this to Printf also.

Reply:
Is this because it is easier to maintain over time? Genuinely curious if Printf() is better than Println() generally. I didn't see anything online about how (in Go specifically) if it is more efficient. I don't mind switching it over, but wondering what I should default to in the future.